src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -377,301 +124,224 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
+        {
             target.Run(new string[] { argName });
 
-            mock.VerifyAll();
+            strykerRunnerMock.VerifyAll();
 
-            actualOptions.DiffEnabled.ShouldBeTrue();
+            inputs.LogToFileInput.SuppliedInput.Value.ShouldBeTrue();
         }
 
         [Theory]
-        [InlineData("--mutation-level")]
-        [InlineData("-level")]
-        public void ShouldSetMutationLevelWhenPassed(string argName)
+        [InlineData("--dev-mode")]
+        public void WithDevModeArgument_ShouldPassDevModeArgumentsToStryker(string argName)
         {
-            var options = new StrykerOptions();
-            var runResults = new StrykerRunResult(options, 0.3);
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>())).Returns(runResults);
+            target.Run(new string[] { argName });
 
-            var target = new StrykerCLI(mock.Object);
+            strykerRunnerMock.VerifyAll();
 
-            target.Run(new string[] { argName, "advanced" });
-            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o =>
-                o.MutationLevel == MutationLevel.Advanced), It.IsAny<IEnumerable<LogMessage>>()));
+            inputs.DevModeInput.SuppliedInput.Value.ShouldBeTrue();
         }
 
         [Theory]
-        [InlineData("--dashboard-compare", "--dashboard-version project")]
-        [InlineData("-compare", "-version project")]
-        public void ShouldEnableDiffCompareToDashboardFeatureWhenPassed(params string[] argName)
+        [InlineData("--concurrency")]
+        [InlineData("-c")]
+        public void WithMaxConcurrentTestrunnerArgument_ShouldPassValidatedConcurrentTestrunnersToStryker(string argName)
         {
-            StrykerOptions options = null;
-            var runResults = new StrykerRunResult(new StrykerOptions(), 0.3);
-
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>()))
-                .Callback<StrykerOptions, IEnumerable<LogMessage>>((c, m) => options = c)
-                .Returns(runResults)
-                .Verifiable();
-
-            var target = new StrykerCLI(mock.Object);
-
-            target.Run(argName);
+            target.Run(new string[] { argName, "4" });
 
-            mock.VerifyAll();
+            strykerRunnerMock.VerifyAll();
 
-            options.CompareToDashboard.ShouldBeTrue();
+            inputs.ConcurrencyInput.SuppliedInput.Value.ShouldBe(4);
         }
 
         [Theory]
-        [InlineData("--dashboard-compare", "--dashboard-version project")]
-        [InlineData("-compare", "-version project")]
-        public void ShouldEnableDiffFeatureWhenDashboardComparePassed(params string[] argNames)
+        [InlineData("--break")]
```

Comment:
This way it reminds rather something boolean, I would suggest something like `--break-threshold`.